apps/web/src/content/blog/2026-03-01.md

```
@@ -0,0 +1,82 @@
+旧サイトにもBlogと銘打たれた場所があったのですが、完全にハリボテのものなのでした。
+（2022年すなわち、4年前。。。）
+![旧サイトのBlog部分](/blog/2026-03-01/image.png)
+
+社会人になって、Zennとかで以下の記事を書いたりなどはしていました。
+- [ISUCONに初参加してみた！(2024. ISUCON14)](https://zenn.dev/jigjp_engineer/articles/55ddffbff7a363)
+- [DBeaverのAIボタン（AI Assistant）](https://zenn.dev/jigjp_engineer/articles/3cdf0c58c98539)
+- [JSpecify 1.0解説：JavaのNull安全性とIntelliJ IDEAの挙動](https://zenn.dev/jigjp_engineer/articles/6f1f5cec8509ca)
+
+しかし、「個人的の思いみたいな場所を語る場」がないなぁと思い、このような場を作成したわけです。もちろん「はてなブログ」等もありだとは思うのですが、完全に自己責任で何でもできる場も作っておきたかったのです。
+
+### 自由に遊べる環境を作りたかった。
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)